docs/alerting/email-destination.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: default
+title: Alerts by email
```

Comment:
Titles are Title Case right now (though we might change later), whereas headers are sentence case.

Reply:
Ok. Will keep that in mind. But right now, this file does not exist.